src/workflows/bidsSpatialPrepro.m

```diff
@@ -44,12 +44,17 @@ function bidsSpatialPrepro(opt)
 
   [BIDS, opt, group] = setUpWorkflow(opt, 'spatial preprocessing');
 
+  opt.orderBatches.selectAnat = 1;
+  opt.orderBatches.realign = 2;
+  opt.orderBatches.coregister = 3;
+  opt.orderBatches.segment = 5;
```

Comment:
Do you think it would be helpful to specify the order of ALL the batches?

Someone seeing this might think: "well this is nice, but what batch is happening in step 4?" 

Or they might think that nothing happens after segmentation. Maybe having the full list of batches and their order could act as a "table of content"?

Reply:
actually I am not sure what these orderBatches do. As a user I was indeed asking for "well this is nice, but what happened to batch 4" :D 

Where do you suggest them to be inserted into?

Reply:
hahaha

OK so basically the `opt.oderBatches` is where we keep track of the order in which the steps of the spatial preprocessing happen. The number for each step indicates the index for the `matlabbatch{index}`.

I will make code suggestions directly to guide you.